cmp/grid/impl/ColumnHeader.js

```diff
@@ -107,8 +108,8 @@ class LocalModel {
     _lastTouchStart = null;
     _lastMouseDown = null;
 
-    constructor({gridModel, xhColumn, column: agColumn, enableSorting}) {
-        this.gridModel = gridModel;
+    constructor({gridLocalModel, xhColumn, column: agColumn, enableSorting}) {
+        this.gridModel = gridLocalModel.model;
```

Comment:
So this was broken before, but we weren't paying a price for it that we noticed?

Why not have the grid pass the actual `GridModel` in as a prop to the header for use directly here? I realize it's internal so am not stressed about it, but having any component need to be aware of another components localModel seems like an anti-pattern, even if they are part of a family.